ex03.ex

```diff
@@ -77,7 +83,8 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+  def list_contains([],_), do: false
+  def list_contains([h|t],x), do: h==x || list_contains(t,x)
```

Comment:
### -1

Your code isn't incorrect, but it is better written as 

~~~ elixir
def list_contains([ x | t ], x), do: list_contains(t, x)
~~~